app/javascript/components/StartEndModal/StartEndModal.jsx

```diff
@@ -38,37 +39,33 @@ const StartEndModal = ({ gameState, startEnd, errors }) => {
   const [selectedTeamId, setSelectedTeamId] = useState(first_team_id || '');
 
   const [open, setOpen] = useState(false);
+  const [teams] = useState(teams_with_players.map((team, index) => {
+      return ( <MenuItem key={index} value={team.team.id}>{team.team.team_name}</MenuItem> );
+    })
+  );
 
   const errorsExist = errors && errors.throw_order;
```

Comment:
The (unused) error state and message labels are probably better candidates for local state